fontc/src/change_detector.rs

```diff
@@ -246,6 +246,22 @@ impl ChangeDetector {
                 .is_file()
     }
 
+    pub fn mark_be_change(&self) -> bool {
+        self.glyph_order_ir_change()
```

Comment:
I think the be marks task also depends on the anchors IR, not only on the glyph order IR, should you not capture this in here?

Reply:
I should!

Reply:
Fixed.